examples/quickstart/main.rs

```diff
@@ -439,24 +454,30 @@ async fn get_tx_requests(
         .await
         .expect("Failed to get nonce");
 
-    let approval_request = TransactionRequest::default()
-        .from(user_address)
-        .to(sell_token_address)
-        .input(TransactionInput { input: Some(AlloyBytes::from(data)), data: None })
-        .gas_limit(50_000u64)
-        .max_fee_per_gas(max_fee_per_gas.into())
-        .max_priority_fee_per_gas(max_priority_fee_per_gas.into())
-        .nonce(nonce);
-
-    let swap_request = TransactionRequest::default()
-        .to(Address::from_slice(&tx.to))
-        .from(user_address)
-        .value(biguint_to_u256(&tx.value))
-        .input(TransactionInput { input: Some(AlloyBytes::from(tx.data)), data: None })
-        .max_fee_per_gas(max_fee_per_gas.into())
-        .max_priority_fee_per_gas(max_priority_fee_per_gas.into())
-        .gas_limit(300_000u64)
-        .nonce(nonce + 1);
+    let approval_request = TransactionRequest{
+        to: Some(TxKind::Call(sell_token_address)),
+        from: Some(user_address),
+        value: None,
+        input: TransactionInput { input: Some(AlloyBytes::from(data)), data: None },
+        gas: Some(100_000u64),
+        chain_id: Some(8453),
+        max_fee_per_gas: Some(max_fee_per_gas.into()),
+        max_priority_fee_per_gas: Some(max_priority_fee_per_gas.into()),
+        nonce: Some(nonce),
+        ..Default::default()
+    };
 
+    let swap_request = TransactionRequest{
+        to: Some(TxKind::Call(Address::from_slice(&tx.to))),
+        from: Some(user_address),
+        value: Some(biguint_to_u256(&tx.value)),
+        input: TransactionInput { input: Some(AlloyBytes::from(tx.data)), data: None },
+        gas: Some(300_000u64),
+        chain_id: Some(8453),
+        max_fee_per_gas: Some(max_fee_per_gas.into()),
+        max_priority_fee_per_gas: Some(max_priority_fee_per_gas.into()),
+        nonce: Some(nonce + 1),
+        ..Default::default()
+    };
```

Comment:
Pain #2: These transaction requests needed to have the proper chain id configured.